src/fiat/funds-transfers.md

```diff
@@ -126,3 +127,113 @@ curl -X GET "https://service.centrapay.com/api/topups" \
   }
 ]
 ```
+
+## Creating a Withdrawl **EXPERIMENTAL**
```

Comment:
Should this be **Withdrawal** instead of Withdrawl

Reply:
Yep, bit of a typo. There's a fix in the pipeline coming!